### PR TITLE
feat: [META-1141] Adding information on SCST - Store SBOM per Build d…

### PR DESCRIPTION
**Pull request by xtreme-jason-smith**
…ata model and concept

### CODE REVIEW

File: scst-store/data-models-and-concepts.hbs.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+This topic describes the different data models and concepts the Supply Chain Security Tools (SCST) - Store uses for consuming and providing various information.
+
+## <a id='overview'></a> Overview
+
+The diagram below shows an overall view of the SCST - Store data models and their relations.
+
+![Screenshot of a SCST -Store Data Model Overview.](images/scst-store-data-model-overview.jpg)
+
+It includes the relations of the following concepts:
+* Software Bill of Material per Build
```
Comment (julia-pu) [inline scst-store/data-models-and-concepts.hbs.md:12]: Small clarification question, does the `It` here refer to the screenshot of the data model overview? Since it seems to contain more than just the SBOM relations

Comment (xtreme-jason-smith) [reply scst-store/data-models-and-concepts.hbs.md:12]: Yes. I was thinking of only listing concepts that this page would go in details. So the diagram will show a bit more, but each section will be a bit more specific.

Does it make it too confusing or?

Comment (julia-pu) [reply scst-store/data-models-and-concepts.hbs.md:12]: I see, maybe reword it in a way that explains that the diagram is not only of relations for sbom per build, but that is just what this page will go into details about? Just thinking as a user that doesn't have context before, this sentence might confuse them into thinking everything in the diagram is related to sbom per build

**Review by julia-pu — APPROVED** [review]

**Review by Pivotal-Jeff-Jun — APPROVED** [review]
